examples/platform/silabs/wifi/WifiInterfaceAbstraction.h

```diff
@@ -18,55 +18,48 @@
 
 #include <app/icd/server/ICDServerConfig.h>
 #include <cmsis_os2.h>
+#include <lib/support/BitFlags.h>
```

Comment:
Not for this pr, but shouldn't we create an interface class for all this?

Reply:
Yes; most of this is Wiseconnect (9116 and 917) specifique but is included by the wf200 for some reason